backend/dataall/modules/mlstudio/cdk/mlstudio_extension.py

```diff
@@ -63,7 +64,7 @@ def extent(setup: EnvironmentSetup):
                     setup,
                     f'SageMakerStudio{_environment.name}',
                     log_group_name=f'/{_environment.resourcePrefix}/{_environment.name}/vpc/sagemakerstudio',
-                    retention=logs.RetentionDays.ONE_MONTH,
+                    retention=getattr(logs.RetentionDays, os.environ.get('LOG_RETENTION', 'TWO_YEARS')),
```

Comment:
In the cdk.json we are expected users to input the log retention time as a string such as `ONE_MONTH`. but the parameter `retention` does not accept strings, it accepts logs.RetentionDays values which are values of an Enum: https://docs.aws.amazon.com/cdk/api/v2/python/aws_cdk.aws_logs/RetentionDays.html